go.mod

```diff
@@ -244,7 +246,7 @@ require (
 	github.com/golang-migrate/migrate/v4 v4.7.0
 	github.com/google/go-github/v45 v45.2.0
 	github.com/grafana/codejen v0.0.3
-	github.com/grafana/dskit v0.0.0-20230202092222-880a7f8141cc
+	github.com/grafana/dskit v0.0.0-20230706162620-5081d8ed53e6
```

Comment:
I downloaded the latest dskit in apiserver go.mod which may be causing this upgrade. I will take a closer look tomorrow.